kotlin-source/build.gradle

```diff
@@ -13,6 +13,15 @@ apply plugin: 'net.corda.plugins.quasar-utils'
 
 jar.baseName = "cordapp-example"
 
+cordapp {
```

Comment:
Should we not make similar sort of change to `build.gradle` under `java-source` ?

Reply:
Yes. I will do it.

Reply:
I added it.